src/algebra/ordered_field.lean

```diff
@@ -181,12 +181,18 @@ namespace nat
 
 variables {α : Type*} [linear_ordered_field α]
 
-lemma inv_pos_of_nat [linear_ordered_field α] {n : ℕ}  : 0 < ((n : α) + 1)⁻¹ :=
+lemma inv_pos_of_nat {n : ℕ} : 0 < ((n : α) + 1)⁻¹ :=
```

Comment:
Can you get rid of the `+1` and add the hypothesis `n ≠ 0`?

Reply:
Sorry for the delay -- I was traveling for a few days. I am still not sure this is a great lemma to have in the library, and anyhow the name would be better as `inv_of_nat_add_one_pos`, but I don't feel strongly about it. I'll merge now.